### ISSUE TITLE
[Keyboard Navigation -  Bot Framework Web Chat- Card Sports] Controls 'Final and SEA' aren't accessible by keyboard under Showing Sports section

### ISSUE THREAD

**Issue by mishraadi** [microsoft/BotFramework-WebChat#4027]
User Experience:
If controls 'Final and SEA' aren't accessible by keyboard under Showing Sports section, then the user will not be able to access and carry out the required actions using those controls.

Note: Login Credentials must not be provided within the bug.

Test Environment:
OS Version: 21H1
OS Build: 19043.1110
Browser: Microsoft Edge Version 94.0.972.2 (Official build) dev (64-bit)
Screen Reader: Narrator

Repro steps:

1. Open URL: https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/01.getting-started/d.es5-direct-line-speech.
2. Web chat bundle will open.
3. Navigate on the web chat bot and select microphone icon.
4. Speak "Card sports" keyword in Type your message box, then select send button.
5. Cards related to the keyword will appear, navigate on the cards.
6. Verify the issue.

Actual Result
Controls 'Final and SEA' aren't accessible by keyboard under Showing Sports section.

Expected Result
Controls 'Final and SEA' should be accessible by keyboard under Showing Sports section.

MAS Reference
MAS 2.1.1-Keyboard

Reference Links
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by mishraadi** [comment]
#A11yMAS;#A11ySev1;#HCL;#MAS2.1.1;#Benchmark;#Accessibility;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
The "Final" and "SEA" is not interactive.

There are only 3 interactable elements in the card:

1. The container including both CAR and SEA
   - ![image](https://user-images.githubusercontent.com/1622400/130520741-b1a346b8-499e-478b-97ff-17915a22ca0a.png)
   - Will open http://msn.com
2. The left column with the CAR and its logo
   - ![image](https://user-images.githubusercontent.com/1622400/130520780-f7ac7ac6-211e-4ed9-8b44-3ea798a25fd6.png)
   - Will open http://so.cl
3. The logo of CAR
   - ![image](https://user-images.githubusercontent.com/1622400/130520811-aee4cfa2-7899-4bf2-b2b5-1b0d9d193828.png)
   - Will open http://msnbc.com

<details>
<summary>Click to expand the Adaptive Cards JSON</summary>

```json
{
  "$schema": "https://microsoft.github.io/AdaptiveCards/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "speak": "The Seattle Seahawks beat the Carolina Panthers 40-7",
  "body": [
    {
      "type": "TextBlock",
      "text": "Seattle vs Panthers",
      "isSubtle": true
    },
    {
      "type": "Container",
      "selectAction": {
        "type": "Action.OpenUrl",
        "url": "http://msn.com",
        "title": "Container action"
      },
      "items": [
        {
          "type": "ColumnSet",
          "columns": [
            {
              "type": "Column",
              "size": "auto",
              "items": [
                {
                  "type": "Image",
                  "url": "https://webchat-mockbot.azurewebsites.net/public/assets/sports-panthers.png",
                  "size": "small",
                  "horizontalAlignment": "center",
                  "selectAction": {
                    "type": "Action.OpenUrl",
                    "url": "http://so.cl",
                    "title": "Image action"
                  }
                },
                {
                  "type": "TextBlock",
                  "text": "CAR",
                  "horizontalAlignment": "center",
                  "size": "normal",
                  "weight": "bolder"
                }
              ],
              "selectAction": {
                "type": "Action.OpenUrl",
                "title": "Column action",
                "url": "http://msnbc.com"
              }
            },
            {
              "type": "Column",
              "size": "stretch",
              "items": [
                {
                  "type": "TextBlock",
                  "text": "Dec 4",
                  "horizontalAlignment": "center"
                },
                {
                  "type": "TextBlock",
                  "text": "Final",
                  "horizontalAlignment": "center"
                },
                {
                  "type": "TextBlock",
                  "text": "7 - 40",
                  "size": "large",
                  "horizontalAlignment": "center"
                }
              ]
            },
            {
              "type": "Column",
              "size": "auto",
              "items": [
                {
                  "type": "Image",
                  "url": "https://webchat-mockbot.azurewebsites.net/public/assets/sports-seahawks.png",
                  "size": "small",
                  "horizontalAlignment": "center"
                },
                {
                  "type": "TextBlock",
                  "text": "SEA",
                  "horizontalAlignment": "center",
                  "size": "normal",
                  "weight": "bolder"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
```
</details>

**Comment by v-shikch** [comment]
We have added this issue in our SME discussion queue. Will take action after getting feedback from SME.

**Comment by compulim** [comment]
Thanks @v-shikch. Look forward to you and SME feedbacks. Feel free to ping me on Teams for faster response.
